dev/AppLifecycle/ActivatedEventArgsBase.h

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 #pragma once
 
-namespace winrt::Microsoft::ApplicationModel::Activation::implementation
+namespace winrt::Microsoft::Windows::AppLifecycle::implementation
```

Comment:
I must be behind the times.  All our API's will now be Microsoft::Windows::something? Solves some issues with us (Windows) owning "Microsoft" top level namespace node. (Microsoft::UI is kind of presumptuous).

Reply:
That - `Microsoft.Windows.*` is correct, and our rule has been to only put things into third-level namespaces like this.